src/ContigInterval.js

```diff
@@ -0,0 +1,41 @@
+/* @flow */
+
+var Interval = require('./Interval');
+
+/**
+ * Class representing a closed interval on the genome: contig:start-stop.
+ *
+ * The contig may be either a string ("chr22") or a number (in case the contigs
+ * are indexed, for example).
+ */
+class ContigInterval {
+  contig: string|number;
```

Comment:
In bigBed format, they use a mapping which converts all the contig names to integers, for more compact storage. So it is convenient to allow `contig` to be either.